apps/edxapp/templates/cms/dc_worker_low_priority.yml.j2

```diff
@@ -0,0 +1,7 @@
+{% extends "dc.yml.j2" %}
+{% block command %}
+        command:
+          - "/bin/bash"
+          - "-c"
+          - python manage.py lms celery worker --loglevel=info --queues=edx.lms.core.high --hostname=edx.lms.core.low.%%h --concurrency=1
```

Comment:
Those 3 templates should target the CMS not the LMS: `python manage.py cms [...] --queues=edx.cms.core.xx --hostname=edx.cms.core.low.%%h [...]`